Includes/menu.cpp

```diff
@@ -223,7 +222,9 @@ void MenuXbe::updateScanningLabel() {
 }
 
 void MenuXbe::onScanCompleted(bool succeeded,
-                              std::vector<XBEScanner::XBEInfo> const& items) {
+                              std::list<XBEScanner::XBEInfo> const& items,
+                              long long duration) {
+  (void)duration;
```

Comment:
Note: I'm piping through duration specifically for ease of debugging, I can pull this out and keep it in a side branch if you'd prefer.